paper-server/patches/sources/net/minecraft/world/entity/EntitySelector.java.patch

```diff
@@ -23,31 +23,49 @@
  
      private EntitySelector() {
      }
-@@ -27,18 +_,24 @@
+@@ -27,29 +_,31 @@
      }
  
      public static Predicate<Entity> pushableBy(Entity entity) {
 +        // Paper start - Climbing should not bypass cramming gamerule
```

Comment:
Comment is not necessary, this method is not touched for the climbing cramming bypass option